### PR TITLE
Eliminate doctor warning about paths with spaces

### PR DESCRIPTION
**Pull request by cbracken**
Some people call me the space cowboy
Some people call me the gangster of love
Some people call me Maurice

### CODE REVIEW

**Comment by cbracken** [comment]
Do not merge yet. Once engine roll (#14612) lands, and Travis is updated so we don't regress this (#14613), this can land.

**Review by goderbauer — APPROVED** [review]
LGTM

**Comment by cbracken** [comment]
Closing #6577!

**Pull request merged**
